src/kruskal.jl

```diff
@@ -0,0 +1,52 @@
+####################################################
+## Minimum Spanning Tree - Kruskal algorithm
+####################################################
+
+"""
+
+Computes the minimum spanning tree of an undirected graph by using Kruskal's algorithm.
+
+Requires arguments:
+distmx::LightGraphs.Graph          # Adjacency matrix of the input graph
+n::Int                             # The number of vertices in the graph 
+
```

Comment:
If we have the graph, we don't need the number of vertices. We can get them via `nv()`